src/test/java/org/takes/facets/auth/social/PsGithubTest.java

```diff
@@ -135,6 +152,17 @@ public void exec(final URI home) throws IOException {
     }
 
     /**
+     * A private method to create the basic directive.
+     * without the access token.
+     * @return A basic directive.
+     */
+    private static Directives getDirectiveWithoutAccessToken() {
```

Comment:
@pinaf  I thought it was very specific directive without the access token, If we rename it to basicDirectives, tomorrow people may confuse?